nebula/src/nebula/transcript/whisper_utils.py

```
@@ -0,0 +1,148 @@
+
+        if not success:
+            raise RuntimeError(
+                f"⚠️ Azure Whisper too many retries for chunk {chunk_path}"
+            )
+
+        offset += get_audio_duration(chunk_path)
+
+    return {"segments": all_segments}
+
+
+def transcribe_with_openai_whisper(audio_path: str, llm_id: str | None = None) -> dict:
```

Pylint (3.3.7)

[refactor] Too many local variables (20/15)

(R0914)